labscommunity/protocol-land
# fix: Prevent closing modal & disable input during form submission

## Summary

This PR introduces enhancements/fixes aimed at improving user experience during form submissions in modals. Key updates include:

- Prevention of modal closure during active form submissions.
- Usage of `cursor-not-allowed` and input disabling during form submission to indicate processing and prevent user interaction.

## How to Test

- **Test different modals:** Perform creating repo, forking repo, uploading new files and more to check if the inputs are disabled and user's can't close the modals during active form submissions.
